webapp/src/locales/en_US.ts

```diff
@@ -0,0 +1,106 @@
+import { LocaleType } from "./index";
+
+const en_US: LocaleType = {
+  Error: {
+    Unauthorized: "Currently not logged in.",
+    MustAuthorized: "Currently not logged in, please log in and use.",
+    Network: "The network has wandered off, please try again later!",
+    LocalCache: "Oops, something went wrong!",
+    LocalCacheTry: "Please clean the cache and try again!",
+    LocalCacheClean: "Clear All Cache",
+  },
+  Dashboard: {
+    QPH: "QPH",
```

Comment:
QPH = Query Per Hour? Then is it average too?

Reply:
It is not an average value because the configuration could be "queryHistoryHoursRetention:1", so there is no design to display average QPH.